### DISCUSSION TITLE
Container Registry inaccessible over IPv6 (DNS)

### DISCUSSION THREAD

**Discussion by sanmai-NL** [community/community#45930] [Other Feature Feedback, Questions, & Ideas]
The Container Registry is unresolvable by AAAA DNS Resource Records.

This excludes IPv6-only users and hampers progress on the Internet. It makes IPv6-only Kubernetes clusters unable to bootstrap in some cases, forcing systems admins to go to great lengths just to get their systems up and running.

See https://www.docker.com/blog/beta-ipv6-support-on-docker-hub-registry/ for what the main competitor has achieved already. GitLab is also working on this. It's not much work and basically done on their end, but out of low change appetite they are lagging behind similarly to GitHub.

**Comment by Arkensor** [comment]
We just came across this in our company too, how the hell is ipv6 not supported by GitHub?
`iPv6 Is tHe fUtUrE`

**Comment by hayschan** [comment]
Same problem. It is hard to believe this happens to GitHub, acquired by one of the largest company on Earth.

**Comment by scolastico** [comment]
I had to set up a SOCKS5 proxy only because GitHub is unable to support a technology that has been around for ~25 years. It's the same kind of fuck-up like when GitHub Actions don't support YAML anchors.

**Comment by lion7** [comment]
I find it unbelievable that a major service like GitHub still does not support IPv6 (which was introduced in 1998).
Feels to me that not supporting IPv6 is almost intentional..
